assets/source/setup-guide/app/style.scss

```diff
@@ -590,6 +590,18 @@ $root: ".woocommerce-setup-guide";
 			display: none;
 		}
 
+		.components-card__footer {
+			flex-direction: row;
+
+			.link {
```

Comment:
```suggestion
			.components-external-link {
```

Reply:
In case we don't need `.link`selector for some reason. I would go for `.components-external-link` default selector

Reply:
Same as https://github.com/woocommerce/pinterest-for-woocommerce/pull/273#discussion_r755826043.